packages/tools/client-debugger/client-debugger-view/src/test/app/index.tsx

```diff
@@ -40,7 +40,7 @@ function DevToolsView(): React.ReactElement {
 				zIndex: "2",
 				backgroundColor: "lightgray", // TODO: remove
 			}}
-			defaultSize={{ width: 400, height: "100%" }}
+			defaultSize={{ width: 500, height: "100%" }}
```

Comment:
Note that this won't affect how the devtools appear in the Chrome extension. Does everything still look okay in that view?

Reply:
Yeah, no issues in the extension view